src/main/java/core/basesyntax/model/User.java

```diff
@@ -8,6 +8,12 @@ public class User {
     private String password;
     private Integer age;
 
+    public User(String login, String password, Integer age) {
```

Comment:
Consider adding a constructor comment to explain the purpose of this constructor, its parameters, and any important details.

Reply:
Sorry, this is an irrelevant comment